keycloak/Pluralfile

```diff
@@ -0,0 +1,6 @@
+REPO keycloak
+ATTRIBUTES phasetwo repository.yaml
```

Comment:
It will unfortunately be a bit tricky to have phasetwo be the publisher in this repo, I think I can set it up so that it will work but generally it uses our internal publishers credentials to run `plural apply` and to vendor images.  You can technically move this folder tree into any repo and replicate our gh actions though.